jib-cli/src/main/java/com/google/cloud/tools/jib/cli/buildfile/Layers.java

```diff
@@ -71,6 +75,10 @@
           Path src = rawSrc.isAbsolute() ? rawSrc : buildRoot.resolve(rawSrc);
           AbsoluteUnixPath dest = copySpec.getDest();
 
+          if (!Files.isDirectory(src) && !Files.isRegularFile(src)) {
```

Comment:
Is this primarily to guard against symlinks?

Reply:
Yeah there's two checks. One for non existence and one for unknown types. Perhaps the messaging should be different